modules/mod_ginger_rdf/models/m_dbpedia.erl

```diff
@@ -80,7 +84,8 @@ task_resource_update(Uri, Language, Context) ->
     end,
     ok.
 
-get_resource_fetch(Uri, Language, StaleData, Context) ->
+get_resource_fetch(Uri0, Language, StaleData, Context) ->
+    Uri = strip_protocol(Uri0),
     Key = cache_key(Uri, Language),
     case dbpedia:get_resource(<<"http://", Uri/binary>>, Language) of
```

Comment:
Why not use the URI with the `http://` as a  key instead?

We already have a `ginger_uri:https` function, we could add a `ginger_uri:http` which forces the protocol to be HTTP and here do something like:
```suggestion
    Uri = ginger_uri:http(Uri0),
    Key = cache_key(Uri, Language),
    case dbpedia:get_resource(Uri, Language) of
```

Reply:
Cleaned up.